cmds/ocm/commands/ocmcmds/common/inputs/types/wget/cli.go

```diff
@@ -0,0 +1,24 @@
+// SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Open Component Model contributors.
```

Comment:
```suggestion
// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Open Component Model contributors.
```

Where is this text coming from? Can we change it centrally?